README.md

```diff
@@ -5,6 +5,19 @@ A command line tool to help with using Continuous Pipe as a remote development e
 This helps to set up Kubectl, create, build and destroy remote environments and keep files
 in sync with the local filesystem.
 
+## Prerequisites
+
+You will need the following:
+
+ * A Continuous Pipe hosted project with the GitHub integration set up
+ * The project checked out locally 
+ * The IP address, username and password to use for Kubenetes cluster
+ * rsync and fswatch installed locally
```

Comment:
it doesn't, it probably can but will be made more complicated by needing to be portable so may not be running on osx and if it is it may not have brew available.